app/views/insured/families/_qle_detail.html.erb

```diff
@@ -1,7 +1,6 @@
 <% if @bs4 %>
   <div class="hidden" id="qle-details">
     <h2> <%= l10n("insured.qle_detail.event_details") %> </h2>
-    <h3 class="qle-details-title"></h3>
```

Comment:
any reason why this is removed? `qle.js` will set the text of this element when the user selects a qle event. without it, no qle title will be displayed

Reply:
Was going to ping about that today: it is causing a WAVE error on the home page, was unsure if it was added mistakenly or if it needs to be modified

Reply:
Discussed offline, reverted back and removed modal from home page https://github.com/ideacrew/enroll/pull/4013/commits/18aed77b2e326f750b7888d1a23f206fca46386d